Client/Frontend/Widgets/SiteTableViewController.swift

```diff
@@ -52,9 +52,13 @@ class SiteTableViewHeader : UITableViewHeaderFooterView {
             make.height.equalTo(0.5)
         }
 
+        // A table view will initialize the header with CGSizeZero before applying the actual size. Hence, the label's constraints
+        // must not impose a minimum width on the content view.
         titleLabel.snp_makeConstraints { make in
-            make.left.equalTo(contentView).offset(SiteTableViewControllerUX.HeaderTextMargin)
-            make.right.equalTo(contentView).offset(-SiteTableViewControllerUX.HeaderTextMargin)
+            make.left.equalTo(contentView).offset(SiteTableViewControllerUX.HeaderTextMargin).priority(999)
```

Comment:
SnapKit includes a `priorityHigh()` -- I think that'd be slightly cleaner than manually defining a priority here.